src/splitInteger.test.js

```diff
@@ -1,17 +1,19 @@
-const splitInteger = require('./splitInteger');
+'use strict';
 
-test(`should split a number into equal parts if a value is divisible by a numberOfParts`, () => {
+const splitInteger = require('./splitInteger');
 
+test(`should split a number into equal 3 if a 6 is divisible by a 2`, () => {
```

Comment:
Im not sure you need to specify numbers in described string. Because it may confuse a user solving this task or you need to write more tests. because this test will be passed either with 4, 8 and with 6 , 12 etc